src/config/dark-sites.config

```diff
@@ -1077,3 +1077,4 @@ zombsroyale.io
 zoomquilt.org
 zoomquilt2.com
 ztdp.ca
+tabstats.com
```

Comment:
This site looks good to me. Please update your Pull Request to maintain the alphabetical order of the file.

Please refer [here](https://github.com/darkreader/darkreader/blob/master/CONTRIBUTING.md#adding-a-website-that-is-already-dark) for further information.

Reply:
Hi apologies, not sure how I missed that, fixed now.

Reply:
Thank you! 😃